package.json

```diff
@@ -14,7 +14,7 @@
     "test": "jest",
     "test:coverage": "jest --coverage",
     "test:routes": "jest -c jest.routes.config.js",
-    "test:a11y": "jest -c jest.a11y.config.js --maxWorkers 15",
+    "test:a11y": "yarn build && jest -c jest.a11y.config.js --maxWorkers 15",
```

Comment:
this should probably be done either as a postinstall hook or in the jenkinsfile in an after build callback i think

Reply:
can we leave it as it is just for now and improve it later on when we get some more time?